tutorials/waiting list api v2/resource/subscribe/token setter/f0b00122-f92d-4d12-8453-e3d083fad9ba.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "tutorials/Waiting list API V2/resource/subscribe/token setter",
```

Comment:
From the namespace of this prototype I expected it to be internal but it is public (reusable). Is this intentional?